dir.props

```diff
@@ -41,6 +42,7 @@
     <!-- Input Directories -->
     <PackagesDir Condition="'$(PackagesDir)'==''">$(ProjectDir)packages/</PackagesDir>
     <ToolsDir Condition="'$(ToolsDir)'==''">$(PackagesDir)Microsoft.DotNet.BuildTools.$(BuildToolsVersion)/lib/</ToolsDir>
+    <NuProjDir Condition="'$(NuProjDir)'==''">$(PackagesDir)NuProj.$(NuProjVersion)/tools/</NuProjDir>
```

Comment:
I see this is where NuProjDir is getting set. I'm not sure if I like the fact this this is required to be setup and restored by all the consuming repo's of BuildTools. We should probably include the library directly in the BuildTools package if we want to depend on it.